src/main.tsx

```diff
@@ -1,14 +1,20 @@
-// eslint-disable-next-line import/default
-import React from 'react'
-// eslint-disable-next-line import/default
-import ReactDOM from 'react-dom/client'
+import { StrictMode } from 'react'
+import { createRoot } from 'react-dom/client'
+import { BrowserRouter } from 'react-router-dom'
 
-import App from './App'
-import './index.css'
+import { Router } from './router'
+import './style/index.scss'
 
-// eslint-disable-next-line import/no-named-as-default-member
-ReactDOM.createRoot(document.getElementById('root') as HTMLElement).render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>
+const root = createRoot(document.getElementById('root') as HTMLElement)
+root.render(
+  <StrictMode>
+    <BrowserRouter>
+      <Router />
```

Comment:
Also, I don't know if we're really going to need a router as the app is not meant as a "web app with user data to persist between page changes".
The desktop app may do well without it.

Although it is a bit too early to know if it is true.

I suggest:
- if the router is actually useless right now (I see that we use it for "showcasing ui blocks") we may remove it and add it later if really needed
- if we expect the router to absolutely be needed by the ui, we may keep it, and remove it later if any refactoring allows it

What do you think?

Reply:
I think the router will be useful for lightening our display logic in React. 

Indeed, it allows to link UI elements to paths and so to rely on it instead of manually handling what what's displayed on the screen with a state.

For example : 
- We could have a layout route on root which renders the sidebar layout that is present across all the app
- We could handle the tab bar logic thanks to paths 

etc.